api/server/collections.ts

```diff
@@ -0,0 +1,37 @@
+import { MongoObservable } from 'meteor-rxjs';
```

Comment:
Maybe splitting this file into multiple files, one for each collection? it might be handy for us later when we'll create the commits for the tutorial